### PR TITLE
styled-components applied

### PR DESCRIPTION
**Pull request by jessicaarrais**

### CODE REVIEW

File: src/components/recipe/recipe-page-ingredients/RecipePageIngredients.tsx [hunk 1]
```diff
@@ -1,5 +1,9 @@
 import React from 'react';
-import './recipe-page-ingredients.css';
+import styled from 'styled-components';
+
+const Ingredient = styled.li`
```
Comment (davicatunda) [inline src/components/recipe/recipe-page-ingredients/RecipePageIngredients.tsx:4]: maybe call ListItem

File: src/components/recipe/recipe-page-instructions/RecipePageInstructions.tsx [hunk 2]
```diff
@@ -1,6 +1,26 @@
 import React from 'react';
 import Icon from '../../icon/Icon';
-import './recipe-page-instructions.css';
+import styled from 'styled-components';
+
+const Instruction = styled.li`
```
Comment (davicatunda) [inline src/components/recipe/recipe-page-instructions/RecipePageInstructions.tsx:5]: maybe call InstructionListItem

File: src/index.tsx [hunk 3]
```diff
@@ -17,7 +17,72 @@ import Icon from './components/icon/Icon';
...
+    font-size: 24px;
+  }
+
+  h3 {
+    font-size: 20px;
+  }
+
+  h4 {
+    font-size: 16px;
+  }
```
Comment (davicatunda) [inline src/index.tsx:61]: this is not going to be needed once you use typography components

**Review by davicatunda — CHANGES_REQUESTED** [review]
I've being thinking, I think we need a naming convention for Styled components. Can you search a little bit on the web how other people are doing this? I found one article: https://medium.com/inturn-eng/naming-styled-components-d7097950a245

But perhaps we can choose our own convention. How about if components that are styled always ended or started with some word? the article uses an object called "S", but we could also not create any object and just name things with a weird characted, like replacing "Likes" with "Likes$Styles" or "S$Likes" or anything that you feel like doing to make clearer that the component is not a component but a style.

On another note, some of this component will eventually be replaced with components from the component Library, the more "real" components we use, the easier it is to have your entire site consistent style wise.

File: src/components/UserProfileButton.tsx [hunk 4]
```diff
@@ -1,6 +1,40 @@
...
+  &:active {
+    background-color: thistle;
+    transform: scale(0.95);
+  }
+`;
+
+const Username = styled.span`
+  color: #000000;
+  margin-left: 8px;
+  font-weight: bold;
```
[the remark this answers is not in the document]
Comment (jessicaarrais) [reply src/components/UserProfileButton.tsx:27]: right! get it!
it was serving a purpose that has been lost.

File: src/components/UserProfileButton.tsx [hunk 5]
```diff
@@ -1,6 +1,40 @@
...
+`;
+
+const AvatarThumbnail = styled.div`
+  margin-left: 8px;
+  width: 32px;
+  height: 32px;
+  overflow: hidden;
+  font-size: 12px;
```
[the remark this answers is not in the document]
Comment (jessicaarrais) [reply src/components/UserProfileButton.tsx:35]: it is resizing the avatar img

File: src/components/cookbook/Cookbook.tsx [hunk 6]
```diff
@@ -2,7 +2,34 @@ import React from 'react';
...
+import styled from 'styled-components';
+
+const Header = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+`;
+
+const CreateRecipeWrapper = styled.div`
+  display: flex;
```
[the remark this answers is not in the document]
Comment (jessicaarrais) [reply src/components/cookbook/Cookbook.tsx:14]: no. I can remove it and reconfigure its sibling div.

File: src/components/cookbook/Cookbook.tsx [hunk 7]
```diff
@@ -2,7 +2,34 @@ import React from 'react';
...
+  margin: 0 -4px;
+`;
+
+const FilterListWrapper = styled.div`
+  margin: 0 4px;
+`;
+
+const SortListWrapper = styled.div`
+  margin: 0 4px;
+`;
```
[the remark this answers is not in the document]
Comment (jessicaarrais) [reply src/components/cookbook/Cookbook.tsx:32]: makes sense

File: src/components/cookbook/Cookbook.tsx [hunk 8]
```diff
@@ -53,10 +80,10 @@ function SortList(props: SortListProps) {
   };
 
   return (
-    <div className="sort-list-container" onClick={handleOnSortList}>
```
[the remark this answers is not in the document]
Comment (jessicaarrais) [reply src/components/cookbook/Cookbook.tsx:56]: wrongly put

File: src/components/UserProfileButton.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (jessicaarrais) [reply src/components/UserProfileButton.tsx:27]: also, I like the S object ideia

File: src/components/ingredient/Ingredient.tsx [hunk 9]
```diff
@@ -30,8 +38,7 @@ function Ingredient(props: IngredientProps) {
   );
 
   return (
-    <li
-      className="ingredient-list"
+    <IngredientList
```
[the remark this answers is not in the document]
Comment (jessicaarrais) [reply src/components/ingredient/Ingredient.tsx:41]: Yes. Maybe I am being excessively repetitive

File: src/components/instruction/Instruction.tsx [hunk 10]
```diff
@@ -2,7 +2,19 @@ import React, { useState } from 'react';
...
+const InstructionList = styled.li`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 32px;
+  margin: 8px 0 8px;
+`;
+const InstructionWrapper = styled.div`
+  width: 100%;
```
[the remark this answers is not in the document]
Comment (jessicaarrais) [reply src/components/instruction/Instruction.tsx:16]: must be reviewed

File: src/components/login-signup/Login.tsx [hunk 11]
```diff
@@ -1,7 +1,35 @@
...
 import { gql, useMutation, useApolloClient } from '@apollo/client';
 import { Button } from '@material-ui/core';
-import './login-signup.css';
+import styled from 'styled-components';
+
+const Card = styled.div`
+  width: 400px;
+  margin: 32px auto;
+  padding: 40px 56px;
+  background-color: #ffffff;
```
[the remark this answers is not in the document]
Comment (jessicaarrais) [reply src/components/login-signup/Login.tsx:10]: colors and design still need to be reviewed/chosen

File: src/components/navigation-bar/NavigationBar.tsx [hunk 12]
```diff
@@ -3,7 +3,34 @@ import { useHistory } from 'react-router-dom';
...
+  z-index: 1;
+  height: inherit;
+  padding: 4px 16px;
+  background-color: #ffccbc;
+  box-shadow: 0 0 4px 0 #bdbdbd;
+`;
+
+const HomeButton = styled.div`
+  display: flex;
+`;
```
[the remark this answers is not in the document]
Comment (jessicaarrais) [reply src/components/navigation-bar/NavigationBar.tsx:29]: it is. the name is wrong. should be 'LeftItems'

File: src/components/login-signup/Login.tsx [hunk 13]
```diff
@@ -1,7 +1,35 @@
...
+`;
+
+const SignupButtonWrapper = styled.div`
+  display: flex;
+`;
+
+const ErrorMessage = styled.p`
+  font-size: 12px;
+  color: red;
+`;
```
[the remark this answers is not in the document]
Comment (jessicaarrais) [reply src/components/login-signup/Login.tsx:32]: it will be implemented in a next diff

**Review by davicatunda — APPROVED** [review]

**Pull request merged**